modules/persistent-storage-csi-secrets-store-driver-install.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-secrets-store.adoc
+//
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-secrets-store-driver-install_{context}"]
+= Installing the Secrets Store CSI Driver
+
+.Prerequisites
+* Access to the {product-title} web console.
+
+* Administrator access to the cluster.
+
+.Procedure
+
+To install the Secrets Store CSI Driver:
+
+. Install the Secrets Store Container Storage Interface (CSI) driver operator:
+.. Log in to the web console.
+.. Click *Operators* → *OperatorHub*.
+.. Locate the Secrets Store CSI Operator by typing Secrets Store CSI in the filter box.
```

Comment:
s/by typing Secrets Store CSI/by typing *Secrets Store CSI*  (or some kind of mark up)